cmd/root.go

```diff
@@ -17,6 +17,9 @@ const (
 	defaultConfigSection = "settings"
 	// defaultTimeoutSecs is the default timeout used for requests to the wakatime api
 	defaultTimeoutSecs = 60
+	// defaultOfflineSync is the default maximum number of heartbeats from the
+	// offline queue, which will be synced upon sending heartbeats to the API.
```

Comment:
Ponctuation at the end is missing in the other comments. Should it be removed from this line?

Reply:
Actually the linter should enforce punctuation at the end of comments. I suspect that the linter does not check comments on non-exported (lower-case) names. Nevertheless, let's keep punctuation and also add it to the other comments.